openshift-hack/e2e/annotate/rules.go

```diff
@@ -34,7 +34,6 @@ var (
 			`\[Feature:GPUDevicePlugin\]`,
 			`\[sig-scheduling\] GPUDevicePluginAcrossRecreate \[Feature:Recreate\]`,
 
-			`\[Feature:ImageQuota\]`,                    // Quota isn't turned on by default, we should do that and then reenable these tests
```

Comment:
Has the quota been turned on by default?

Reply:
No, this is an origin test, see https://github.com/openshift/origin/blob/cd282f24b7e94261b0bfaf51fa6265eb6e1799a6/test/extended/imageapis/quota_admission.go#L31 and https://github.com/openshift/origin/blob/cd282f24b7e94261b0bfaf51fa6265eb6e1799a6/test/extended/imageapis/quota_admission.go#L31 so it should live there, not in o/k.

The general rule is that o/k annotation rules live in this repo, whereas o/o rules in origin.